### PR TITLE
Add SASS version

### PR DESCRIPTION
**Pull request by FinalAngel**
This PR replaces ``less`` with ``sass``.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,18 +6,17 @@
   "author": "Start Bootstrap",
   "license": {
     "type": "MIT",
-    "url": "https://github.com/BlackrockDigital/startbootstrap/blob/gh-pages/LICENSE"
+    "url": "https://github.com/divio/startbootstrap-clean-blog"
```
Comment (vxsx) [inline package.json]: if you do this, then you should also change name, title and version (if we are not publishing this on npm, then add private: true as well)

File: sass/clean-blog.scss [hunk 2]
```diff
@@ -454,5 +455,5 @@ img::-moz-selection {
 }
 
 body {
-	webkit-tap-highlight-color: @brand-primary;
-}
\ No newline at end of file
+	webkit-tap-highlight-color: $brand-primary;
```
Comment (vxsx) [inline sass/clean-blog.scss]: 100% sure there should be a dash here

Comment (FinalAngel) [inline sass/clean-blog.scss]: thats how it is in the theme, will not change files to be in sync with upstream

File: package.json (hunk 1 above)
Comment (FinalAngel) [inline package.json]: added private: true

**Comment by vxsx** [comment]
:+1:

**Pull request merged**